file_format.md

```diff
@@ -235,3 +243,55 @@ JSON files accepted by the simularium-viewer contain the following data in JSON
     }
 }
 ```
+
```

Comment:
added spec for binary files, which we discussed here https://github.com/simularium/simularium-viewer/issues/123